CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Owners of Apache MXNet
 
 # Global owners
-*			@piiswrong @mli
+*			@piiswrong @mlii @nswamy @pluskid @codingcat @zihaolucky @qkou @terrytangyuan @hotpxl @hliu @skm @tqchen @ziheng @sxjscience @jspisak @ldpe2g @the @liuyizhi @muli @bingxu @gengyifeng @yzhang87 @ozawa @jxie @cjolivier01 @indhub @lxn2 @yajiedesign @jianzhangzju @terrychen @haibin @madjam @zhasheng
```

Comment:
Are these supposed to be GitHub ids? Many of these are not. e.g. `mlii`, `ziheng `, `bingxu`, `haibin`, `zhasheng`

Reply:
They are supposed to be github id AFAIK.
I've picked the names and id from http://incubator.apache.org/projects/mxnet.html

Reply:
Looks like that was Apache id (or not sure what ids are those).

Reply:
You can use @apache/mxnet-committers instead, so that new committers won't need to be maintained. Also, the file can be moved to `.github` folder. https://github.com/blog/2392-introducing-code-owners